lib/kv-redis.js

```diff
@@ -42,10 +63,30 @@ function RedisKeyValueConnector(settings, dataSource) {
 
 util.inherits(RedisKeyValueConnector, Connector);
 
+/**
+ * Returns "PONG". Used to verify connectivity to the data source.
+ *
+ * @callback {Function} callback
+ * @param {Error} err Error object.
+ * @param {String} message The "PONG" message string.
+ *
+ * @header RedisKeyValueConnector.ping(cb)
```

Comment:
I'm fine with this, but if so, please open one or more issues to make the corresponding change everywhere else.